src/deroff.rs

```diff
@@ -657,82 +657,91 @@ impl Deroffer {
         unimplemented!()
     }
 
+    fn prch(&self, idx: usize) -> bool {
+        self.s.get(idx..(idx+1))
```

Comment:
I believe the parenthesis is not needed here? Maybe we need to clippy clean the file later?

Reply:
You're right, I should have done `(idx..=idx)`, I ran `fmt` and `clippy`: ea0c2fe